front/tests/021-get-manchette.spec.ts

```diff
@@ -0,0 +1,261 @@
+);
+test.describe('GET & Manchette', () => {
+  test.slow();
+  test.use({ viewport: { width: 1920, height: 1080 } });
+
+  let simulationResultPage: OpSimulationResultPage;
+  let scenarioTimetableSection: ScenarioTimetableSection;
+  let scenarioPage: ScenarioPage;
+  let pacedTrainSection: PacedTrainSection;
+  let getManchetteComponent: GetManchetteComponent;
+
+  let project: Project;
+  let study: Study;
+  let scenarioItems: Scenario;
+  let infra: Infra;
+
+  test.beforeAll('Fetch project, study and infrastructure', async () => {
+    project = await getProject(timetableItemProjectName);
+    study = await getStudy(project.id, timetableItemStudyName);
+    infra = await getInfra();
+    scenarioItems = (
+      await createScenario(
+        generateUniqueName('edit-train-scenario'),
+        project.id,
+        study.id,
+        infra.id
+      )
+    ).scenario;
+    await sendTrainSchedules(
+      scenarioItems.timetable_id,
+      JSON.parse(JSON.stringify(trainSchedulesJson.slice(20, 21)))
+    );
+    await sendPacedTrains(
+      scenarioItems.timetable_id,
+      JSON.parse(JSON.stringify(pacedTrainsJson.slice(6, 7)))
+    );
+  });
+
+  test.beforeEach('Open scenario and wait for infra to be loaded', async ({ page }) => {
+    [
+      simulationResultPage,
+      scenarioTimetableSection,
+      scenarioPage,
+      pacedTrainSection,
+      getManchetteComponent,
+    ] = [
+      new OpSimulationResultPage(page),
+      new ScenarioTimetableSection(page),
+      new ScenarioPage(page),
+      new PacedTrainSection(page),
+      new GetManchetteComponent(page),
+    ];
+
+    await page.goto(
+      `/operational-studies/projects/${project.id}/studies/${study.id}/scenarios/${scenarioItems.id}`
+    );
+    await scenarioPage.removeViteOverlay();
+    await waitForInfraStateToBeCached(infra.id);
+  });
+
+  test.afterAll('Delete the created scenario', async () => {
+    await deleteScenario(project.id, study.id, scenarioItems.name);
+  });
+
+  test('Basic GET/Manchette', async () => {
+    await test.step('Project train schedule', async () => {
+      await scenarioTimetableSection.projectTrain(0);
+      await scenarioPage.toggleTrainList();
+    });
+    await test.step('Assert GET slider', async () => {
+      await getManchetteComponent.assertDefaultSliderValue();
+    });
+
+    await test.step('Toggle linear km mode', async () => {
+      await getManchetteComponent.toggleLinearKmMode();
+    });
+
+    await test.step('Expand and collapse warped map', async () => {
+      await getManchetteComponent.expandWarpedMap();
+      await getManchetteComponent.collapseWarpedMap();
+    });
+
+    await test.step('Zoom controls on GET', async () => {
+      await getManchetteComponent.adjustAndResetGetZoom();
+    });
+
+    await test.step('Zoom controls on Manchette', async () => {
+      await getManchetteComponent.zoomInAndResetManchette();
+    });
+  });
+
+  test('GET', async () => {
```

Comment:
```suggestion
  test('STD', async () => {
```
or just `spacetimediagram` if you want to be more explicit